src/pages/public/account/LoginPage.vue

```diff
@@ -1,3 +1,61 @@
 <template>
-    <h1>This is Login Page</h1>
-</template>
\ No newline at end of file
+    <div class="container">
+        <h1 class="headline">Hi there! <br> <span class="headline--accent">Care to log in?</span></h1>
+        <form-card>
+            <template #form-controls>
+                <base-input v-model="email" @blur="validateEmail" inputLabel="email" :isValid="isEmailError" placeholder="Enter your email"/>
+                <password-input v-model="password" />
+            </template>
+            <template #form-actions>
+                    <base-button :block="true">
+                        <router-link :to="{name: 'register'}">
+                            Register instead
+                        </router-link>
+                    </base-button>
+                    <base-button :block="true" buttonType="primary">Log in</base-button>
```

Comment:
remember to always define button `type` inside a `<form>` (unless there is just one and it is submit button)